### PR TITLE
new puppet 4 code

### PR DESCRIPTION
**Pull request by neilmillard-msm**
spec version 4.8.2

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -314,30 +314,49 @@ clone_git_repo() {
 }
 
 # Symlink the cloned git repo to the usual location for Puppet to run
+# Puppet 4 directories
+# mkdir /etc/puppetlabs
+# ln -s /opt/dwh-role-configs/puppet /etc/puppetlabs/puppet
```
Comment (jimfdavies) [inline init.sh]: Too much sharing about internals?

Comment (neilmillard-msm) [inline init.sh]: yes, corrected

**Comment by neilmassey** [comment]
:+1: 
instead of using a different branch, could you add a 'switch' in the init to use 3 or 4?

**Comment by neilmillard-msm** [comment]
It could be implemented as a switch, that would mean updating the Rightscripts too, so they know about the switch, and further config in the control repo to pass the switch instead of the Branch.

**Pull request closed without merging**